tests/wasi-wast/wasi/tests/path_rename.rs

```diff
@@ -162,6 +244,7 @@ fn run_with_different_sub_dirs() {
 
 fn main() {
     run_with_toplevel_dir();
+    run_with_toplevel_dir_overwrite();
     run_with_sub_dir();
     run_with_different_sub_dirs();
 }
```

Comment:
Out of curiosity, why do we use a `main()` function that invokes each test sequentially instead of Rust's builtin test harness?

Reply:
Because the test is ran both as native and wasm, and the native ran output is used to build the actual .wast test (to build the expected behaviour automaticaly)